sites/shared/components/sets/measie-input.mjs

```diff
@@ -0,0 +1,209 @@
+import { isDegreeMeasurement } from 'config/measurements.mjs'
+import { measurementAsMm, formatMm } from 'shared/utils.mjs'
+import { Collapse } from 'shared/components/collapse.mjs'
+import { PlusIcon, EditIcon } from 'shared/components/icons.mjs'
+import { useState } from 'react'
+export const ns = ['account']
+
+const Mval = ({ m, val = false, imperial = false, className = '' }) =>
+  val ? (
+    isDegreeMeasurement(m) ? (
+      <span className={className}>{val}°</span>
+    ) : (
+      <span
+        dangerouslySetInnerHTML={{ __html: formatMm(val, imperial ? 'imperial' : 'metric') }}
+        className={className}
+      />
+    )
+  ) : null
+
+const heightClasses = {
+  2: 'h-12',
+  4: 'h-10',
+  8: 'h-8',
+  16: 'h-6',
+  32: 'h-4',
+}
+const fractionClasses =
+  'w-full border-2 border-solid border-base-100 hover:border-secondary bg-secondary rounded bg-opacity-50 hover:bg-opacity-100'
+const FractionButtons = ({ t, fraction }) => (
```

Comment:
Surprised to see there's no empty line before this method. Do you not have the prettier pre-commit hook setup?

Reply:
In any case, please leave an empty line between methods